main.rb

```diff
@@ -51,13 +30,16 @@ def h(text)
 end
 
 post '/memos' do
-  memo = { 'id' => build_id, 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }
-  save_memo(memo['id'], memo)
-  redirect "/memos"
+  id = memo.build_id
+  title = params['title']
+  content = params['content']
+  created_at = Time.now
+  memo.create(id, title, content, created_at)
```

Comment:
項目が増えるとその都度引数が増えていってしまうので、paramsを直接渡すようにしてしまった方がいいかも

```suggestion
  memo.create(params)
```